source/scenes/Rsvp/Panes/LanguageInfo.jsx

```diff
@@ -11,8 +11,8 @@ const LanguageInfo = () => (
     {({ data, errors, registerField }) => (
       <div className="scrolled-form language-info">
         <div className="instructions">
-          <h2>Please rate your experience with each language from 0-10</h2>
-          <h3>0 - No Experience, 5 - Some Experience, 10 - Very Comfortable</h3>
+          <h2>Please rate your experience with each language from 1-10</h2>
+          <h3>1 - No Experience, 5 - Some Experience, 10 - Very Comfortable</h3>
```

Comment:
Is it feasible to add the missing value to the sliders? 5 is no longer the midpoint between the two extrema.